components/tooltip/__examples__/button-group.jsx

```diff
@@ -12,35 +12,44 @@ class Example extends React.Component {
 	render() {
 		return (
 			<IconSettings iconPath="/assets/icons">
-				<ButtonGroup className="slds-p-bottom_medium">
+				<ButtonGroup className="slds-p-bottom_medium" variant="list">
 					<Tooltip
 						id="refresh-tooltip"
 						align="bottom"
 						content="Buttonbar Tooltip"
+						variant="list-item"
 					>
 						<Button id="refresh-button" label="Refresh" />
 					</Tooltip>
 					<Tooltip
 						id="edit-tooltip"
 						align="bottom right"
 						content="Buttonbar Tooltip"
+						variant="list-item"
 					>
 						<Button label="Edit" id="edit-button" />
 					</Tooltip>
-					<Dropdown
-						id="options"
-						assistiveText={{ icon: 'More Options' }}
-						buttonVariant="icon"
-						iconCategory="utility"
-						iconName="down"
-						iconVariant="border-filled"
-						onSelect={(item) => console.log('selected', item)}
-						options={[
-							{ label: 'A Option', value: 'A0' },
-							{ label: 'B Option', value: 'B0' },
-							{ label: 'C Option', value: 'C0' },
-						]}
-					/>
+					<Tooltip
```

Comment:
Tooltip isn't working for me here. You are welcome to just remove it if you want. I'm guessing this is a side affect of Dropdown.

Reply:
Removing the tooltip is adding double borders ( no overlap ) and is also not adding the `corner radius` to dropdown button.

Reply:
I am wrapping it in an `li` tag for now.

Reply:
The error is being generated from here, we can't be adding props to DOM elements in `camelCase`, they need to be in `lowercase`.

So, it would be best if we can either type check the element and filter out `li` or let `Tooltip` be here unless it is creating problems.

@interactivellama

Reply:
A tooltip around a Dropdown is fine. My issue is that the Tooltip wasn't being triggered on mouseover / "not working". I was thinking that if test the displayName === Tooltip then the extra props wouldn't get passed unless Tooltip was present. I might be misunderstanding though.

Reply:
I am reverting to a tooltip around the dropdown instead of li. I will also open another issue for the Tooltip not being triggered and we can work on it there.